src/main/java/frc/robot/subsystems/DiferentialDrive.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.subsystems;
```

Comment:
This file is unneeded (it's just a spelling thing)

Reply:
deleted!